File: Lecture 3: If-Else, While loop & Lots of Patterns/Pattern5.java
package com.company;

import java.util.Scanner;

public class Pattern5 {
    // Pattern 5
    public static void main(String [] args){
        /*
        Pattern 5
          *
          * *
          * * *
          * *
          *
         */
        // Creating Scanner Object For taking inputs from the USER
        Scanner sc = new Scanner(System.in);
        // Take Number of ROWS and COLUMNS from the USER
        System.out.print(" Enter Number of ROWS for the Pattern: ");
        int n = sc.nextInt();
        pattern5(n);
    }
    static void pattern5(int n){
        // Procedure to print the Required Pattern
        for(int row =0; row < 2*n ; row++){
            int totalColsInRow = row > n ? 2 * n - row : row ;
            for(int column = 0; column < totalColsInRow; column ++){
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}

File: Lecture004 Solving Pattern Questions (Part-2)/pattern5.py

class solution():
    def pattern(n):
        row=1
        while(row<=n):
            col,value=1,row
            while(col<=row):
                print(value,end= " ")
                value+=1
                col+=1
            print()
            row+=1

            

if __name__ == '__main__':
    ob=solution
    num = int(input("Enter Number = "))
    ob.pattern(num)